testing/src/main/java/io/stargate/it/grpc/streaming/ExecuteBatchStreamingTest.java

```diff
@@ -116,6 +117,53 @@ public void onCompleted() {}
                     rowOf(Values.of("c"), Values.of(3)))));
   }
 
+  @Test
+  public void manyStreamingBatch(@TestKeyspace CqlIdentifier keyspace) {
+    List<StreamingResponse> responses = new CopyOnWriteArrayList<>();
+
+    StargateStub stub = asyncStubWithCallCredentials();
+    StreamObserver<StreamingResponse> responseStreamObserver =
+        new StreamObserver<StreamingResponse>() {
+          @Override
+          public void onNext(StreamingResponse value) {
+            responses.add(value);
+          }
+
+          @Override
+          public void onError(Throwable t) {}
+
+          @Override
+          public void onCompleted() {}
+        };
+
+    StreamObserver<Batch> requestObserver = stub.executeBatchStream(responseStreamObserver);
+
+    int queries = RandomUtils.nextInt(10, 50);
```

Comment:
Might want to just use static count here just because otherwise different runs have different count... and while over time this may cover more cases it makes test runs less reproducible?